### PR TITLE
Announce Tenzir Node v4.24

### PR DESCRIPTION
**Pull request by dominiklohmann**
The usual. Likely to be released on Monday.

### CODE REVIEW

File: web/blog/tenzir-node-v4.24/index.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+
+We've completed the migration of all connectors and formats to TQL2, making them
+more powerful and easier to use than ever. Every connector and format now fully
+supports TQL2's expressive syntax and features. At this point, just a handful of
+operators remain to be migrated to TQL2: `chart`, `deduplicate`, `lookup`,
+`parse`, and `print`. We're working hard to complete this migration, but think
+that at this point we're already covering 99% of our use cases with TQL2.
+
+In addition to the above, the release also contains numerous quality-of-life
+improvemnts and bug fixes, so be sure to check out the [changelog][changelog].
```
[the remark this answers is not in the document]
Comment (dominiklohmann) [reply web/blog/tenzir-node-v4.24/index.md:210]:
```suggestion
improvements and bug fixes, so be sure to check out the [changelog][changelog].
```

**Review by mavam — APPROVED** [review]
Flawless.

File: web/blog/tenzir-node-v4.24/index.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+
+- **Lookup Tables** excel at key-value mappings with features like subnet
+  matching and value aggregation
+- **Bloom Filters** provide space-efficient set membership testing
+- **GeoIP Databases** offer specialized geographic information lookup for IP
+  addresses
+
+We've made contexts first-class citizens in TQL2 with robust APIs for creation,
+updates, and enrichment operations. At this point, only the `lookup` operator is
+missing in TQL2.
```
Comment (mavam) [inline web/blog/tenzir-node-v4.24/index.md:195]: I would remove the `lookup` part, as you're also describing below what's missing. Once is enough.

Comment (dominiklohmann) [reply web/blog/tenzir-node-v4.24/index.md]: Good call. I removed the entire paragraph, as the rest of it didn't make much sense without this.

**Pull request merged**